Clarify Azure Front Door move support (Classic vs Standard/Premium); add note and links (#127781)
Azure/azure-content: articles/azure-resource-manager/management/move-support-resources.md

> | edgenodes | No | No | No |
> | profiles | **Yes** | **Yes** | No |
> | profiles / endpoints | **Yes** | **Yes** | No |

> >

Reviewer: ```suggestion

```

> > > [!IMPORTANT]

Reviewer: ```suggestion
> [!IMPORTANT]
```

> > **Azure Front Door Standard/Premium** objects (profiles, endpoints, routes, WAF policies) appear under the **Microsoft.Cdn** provider. These support move across **resource groups** and **subscriptions**. Region moves aren’t supported. See the FAQ and **Networking move guidance** for prerequisites. 

Reviewer: ```suggestion
> **Azure Front Door Standard/Premium** objects (profiles, endpoints, routes, WAF policies) appear under the **Microsoft.Cdn** provider. The support move across **resource groups** and **subscriptions**. Region moves aren't supported. See the FAQ and **Networking move guidance** for prerequisites. 
```

> | vpnserverconfigurations | No | No | No |
> | vpnsites (Virtual WAN) | No | No | No |

> [!NOTE]

> > **Azure Front Door Standard/Premium** resources are provided under the **Microsoft.Cdn** resource provider (see the **Microsoft.Cdn** section below). These **support moves between resource groups and subscriptions**.  

Reviewer: ```suggestion
> **Azure Front Door Standard/Premium** resources are provided under the **Microsoft.Cdn** resource provider (see the **Microsoft.Cdn** section below). The **support moves between resource groups and subscriptions**.  
```

> > If you’re using **Front Door (classic)** (`Microsoft.Network/frontdoors`), **migrate to Standard or Premium first**, then perform the move.  

Reviewer: ```suggestion
> If you're using **Front Door (classic)** (`Microsoft.Network/frontdoors`), **migrate to Standard or Premium first**, then perform the move.  
```

> > See: [Front Door FAQ — move support](../../frontdoor/front-door-faq.yml#can-i-move-front-door-and-cdn-profiles-between-resource-groups-or-subscriptions-without-any-downtime-) and [Migrate Front Door (classic) to Standard/Premium](../../frontdoor/migrate-tier.md).

Reviewer: ```suggestion
> For more information, see [Front Door FAQ — move support](../../frontdoor/front-door-faq.yml#can-i-move-front-door-and-cdn-profiles-between-resource-groups-or-subscriptions-without-any-downtime-) and [Migrate Front Door (classic) to Standard/Premium](../../frontdoor/migrate-tier.md).
```
